src/test/java/Homework20.java

```diff
@@ -0,0 +1,45 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+import org.testng.annotations.Parameters;
+import org.testng.annotations.Test;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+
+import java.time.Duration;
+
+public class Homework20 extends BaseTest {
+    //1. Create a new file Homework20.java with @Test annotation in IntelliJ IDEA.
+    //2. Refactor your code - implement Explicit waits.
+    //3. Remove all implementations of Thread.sleep.
+    //4. Make sure your code runs without failures.
+    //5. Create a new branch and commit your changes.
+    //6. Push your code to a remote repository.
+    //7. Create a pull request.
+
+    @Test
+    @Parameters({"baseURL"})
+    public void deletePlaylist() throws InterruptedException {
```

Comment:
Please remove the "throws InterruptedException" from the code ( since the method does not throw it after you removed a Thread.sleep())

Reply:
Sounds good. I will remove it.